src/components/policy/test/sql_pt_representation_test.cc

```diff
@@ -1038,6 +1038,10 @@ TEST(SQLPTRepresentationTest3,
   SQLPTRepresentation reps(
       kStorageFolder, kAttemptsToOpenPolicyDb, kOpenAttemptTimeoutMs);
   EXPECT_EQ(::policy::SUCCESS, reps.Init(&policy_settings_));
+  if (reps.db()->Open()) {
```

Comment:
@okozlovlux maybe you need something like `IsOpened()`? Because `Open()` call should try to open database.